labs/iot/gps-lab/steps/add-gps-to-pi.md

```diff
@@ -0,0 +1,19 @@
+# Add the GPS sensor to Raspberry Pi
+
+In this step you will set up the GPS sensor to Pi.
+
+## Steps
+
+1. Put the vcc to 2, ground to 6 tx and rx to 8 and 10
+![Pin Reference](../images/raspberry-pi-pins.jpg)
+2. The GPS Module connected should look like following with a blinking red light when the satelite is available
+![Pi with GPS](../images/raspberry-pi-connected.jpg)
+3. Note that the GPS antenna should face the sky to get the GPS values
+
+## Run the Python Code
+1. Connect the Raspberry Pi to the computer
+2. Open the Python Editor (Default Thonny) or Visual Studio Code
+3. Run raspberry-pi-gps-tracker.py
```

Comment:
How? Where is this file? If it's from this repo, how do they get it onto the Pi? Should they work though coding it up so they have an understanding of what the code does? What requirements does the file have for additional libraries, and how are they installed?